parcel_sourcemap/src/lib.rs

```diff
@@ -236,17 +236,13 @@ impl SourceMap {
         result_vec
     }
 
-    pub fn get_source_index(&self, source: &str) -> Result<Option<u32>, SourceMapError> {
+    pub fn get_source_index(&self, source: &str) -> Option<u32> {
```

Comment:
Technically a breaking API change?

Reply:
Right... totally forgot we published this crate, doing a major version for this seems a bit much, will undo it